sixtyfps_runtime/corelib/graphics/image.rs

```diff
@@ -11,6 +11,175 @@ LICENSE END */
 use crate::slice::Slice;
 use crate::{SharedString, SharedVector};
 
+/// SharedPixelBuffer is a container for storing image data as pixels, backed by
+/// [`SharedVector`]. That means it is internally reference counted and cheap
+/// to clone.
+///
+/// You can construct new a new empty shared pixel buffer with [`SharedPixelBuffer::new`],
```

Comment:
```suggestion
/// You can construct new a new shared pixel buffer with [`SharedPixelBuffer::new`],
```

Empty would mean the image is empty and that's not so useful. I wanted to replace empty by transparent but that's not right either

Reply:
I think it's a good idea to mention that the image will be empty - it visually is. Otherwise one might wonder if the pixel data is undefined/uninitialized.

Or does "empty" for you mean zero width and height (despite `new` taking those parameters)?